### ISSUE TITLE
Generate a client library export from OpenAPI specification

### ISSUE THREAD

**Issue by Phoscur** [ugml/api#53]
Generate an (optionally minified) client.js lib file based on the OpenAPI spec including a family jsdoc comments (see related #27, #28).

- client.js
- jsdoc
- client.min.js

**Comment by Phoscur** [comment]
The lib can be used to drive E2E/integration test runs, including full blown database connections, which the unit tests should not.

**Comment by Phoscur** [comment]
Question: should the generated client.js have a less restrictive license like MIT or BSD?

**Comment by mamen** [comment]
What would the purpose of this client-lib be? Providing functions to interact with the API?

**Comment by Phoscur** [comment]
Yep, it doesn't even need to bringt it's own AJAX-request method, it could inject something axios like. It's a collection of request methods with jsdoc e.g.
```javascript
ugmlAPI(request).getPlanets(userID):
// and/or
const api = ugmlAPI(axios);
const player = api.postCreatePlayer(data);
...
```

**Comment by Phoscur** [comment]
The purpose is clearly to make use of our documentation and make it easier to use the API.

**Comment by Phoscur** [comment]
> What would the purpose of this client-lib be? Providing functions to interact with the API?

Actually I dont't think this would work or make sense xD
